README.md

```diff
@@ -51,57 +51,32 @@ After you're finished please remove all the comments and instructions!
   - [Run tests](#run-tests)
   - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
```

Comment:
- [ ] Good job implementing the correct README as suggested by the reviewer 👍👍. However, while doing that, please make sure that you include all the sections/headers in your README according to the template provided by Microverse. As an example, you can see [this README](https://github.com/KanzaTahreem/Blog-App), but please make sure that the content is customized according to the project you are working on 🥇⚒
- [ ] Also, please make sure to remove the template logo. The logo is for Microverse README template, however while using the template, please make sure to customize it. Optionally you can use your own logo. 💪